applications/ray/variables.tf

```diff
@@ -183,3 +183,28 @@ variable "goog_cm_deployment_name" {
   type    = string
   default = ""
 }
+
+# These default resource quotas are set intentionally high as an example that won't be limiting for most clusters.
+# Consult https://kubernetes.io/docs/concepts/policy/resource-quotas/ for additional quotas that may be set.
+variable "resource_quotas" {
```

Comment:
What will be the default value if we don't set this quota. I think our project only have 16 GPU as the quota. Will 100 be too more?

Reply:
This only acts as an upper limit to the number of GPUs Ray namespace Pods can request in their manifests. It won't interact with GCP resource quotas at all and is set intentionally high as an example that won't limit most workloads.